### ISSUE TITLE
If original filename is in lowercase, generate new file in lowercase

### ISSUE THREAD

**Issue by diegohaz** [diegohaz/generact#5]
Currently, if we find `home.jsx`, for example, and replicate it as `Foo`, the new file will be `Foo.jsx`, instead of `foo.jsx`.

Boilerplate that uses that pattern: https://github.com/jedireza/aqua

**Comment by joao-alberto** [comment]
Hi, your project is awesome, congrats!

Someone is assigned to this issue? I wanna contribute 🤘

**Comment by diegohaz** [comment]
@psicotropicos Sure, go ahead, please. :)

**Comment by marlonmleite** [comment]
@diegohaz with this PR the correct is:

**before**
`foo-name.jsx = FooName.jsx`
`foo.jsx = Foo.jsx`

**after**
`foo-name.jsx = foo-name.jsx`
`foo.jsx = foo.jsx`

**Comment by diegohaz** [comment]
@marlonmleite Yeah, but component name (in file contents) should be kept as `FooName` and `Foo`.
